docs/ingestion/native-batch.md

```diff
@@ -387,19 +400,44 @@ them to create the final segments. Finally, they push the final segments to the
 > the task may fail if the input changes in between the two passes.
 
 #### Multi-dimension range partitioning
-> Multiple dimension (multi-dimension) range partitioning is an experimental feature. Multi-dimension range partitioning is currently not supported in the sequential mode of the Parallel task.
 
-When you use multi-dimension partitioning for your data, Druid is able to distribute segment sizes more evenly than with single dimension partitioning.
+> Multiple dimension (multi-dimension) range partitioning is an experimental feature.
+> Multi-dimension range partitioning is currently not supported in the sequential mode of the
+> Parallel task.
 
-For segment pruning to be effective and translate into better query performance, you must include the first of your `partitionDimensions` in the `WHERE` clause at query time. For example, given the following `partitionDimensions`:
-```
- "partitionsSpec": {
-        "type": "range",
-        "partitionDimensions":["coutryName","cityName"],
-        "targetRowsPerSegment" : 5000
+When you use multi-dimension range partitioning for your data, Druid is able to distribute segment
+sizes more evenly than with single dimension partitioning.
+
+Range partitioning has several benefits:
```

Comment:
Consider combining lines 348-364 and 411-440 and moving the text into a separate section titled **Benefits of range partitioning**.

Reply:
> Consider combining lines 348-364 and 411-440 and moving the text into a separate section titled **Benefits of range partitioning**.

I made this change.

> Isn't type of the dimensions always string (atleast for now)?
> Also, the pruning happens assuming a RangeSet of String using string comparisons only, even for equality.

I made this change too. @kfaraz please let me know if the new text is correct.